libcontainer/rootfs_linux.go

```diff
@@ -748,6 +748,41 @@ func pivotRoot(rootfs string) error {
 }
 
 func msMoveRoot(rootfs string) error {
+	mountinfos, err := mount.GetMounts()
+	if err != nil {
+		return err
+	}
+
+	absRootfs, err := filepath.Abs(rootfs)
```

Comment:
AFAICS this is not needed since rootfs is already validated by `(*ConfigValidator).rootfs()`